changelog.txt

```diff
@@ -1,9 +1,9 @@
 0.83.1
  * Added Realm.isClosed() method.
- * Added RealmQuery.isValid(), RealmResults.isValid() and RealmList.isValid(). Each method checks whether the instance is still valid to use or not.
- * IllegalStateException is now thrown instead of native crash if following methods in RealmList are called after closing the Realm or removing parent object: add(int,RealmObject), add(RealmObject),
- * IllegalStateException is now thrown instead of ArrayIndexOutOfBoundsException if following methods in RealmList are called after closing the Realm or removing parent object: set(int,RealmObject), move(int,int), remove(int), get(int)
- * IllegalStateException is now thrown instead of returning 0/null if following methods in RealmList are called after closing the Realm or removing parent object: clear(), removeAll(Collection), remove(RealmObject), first(), last(), size(), where()
+ * Added RealmQuery.isValid(), RealmResults.isValid() and RealmList.isValid(). Each method checks whether the instance is still valid to use or not(for example, the Realm has been closed or any parent object has been removed).
+ * Following methods in RealmList now throw IllegalStateException instead of native crash when RealmList.isValid() returns false: add(int,RealmObject), add(RealmObject),
+ * Following methods in RealmList now throw IllegalStateException instead of ArrayIndexOutOfBoundsException when RealmList.isValid() returns false: set(int,RealmObject), move(int,int), remove(int), get(int)
```

Comment:
This is only true for managed RealmLists. un-managed RealmLists don't throw these exceptions